lib/api/entities.rb

```diff
@@ -66,6 +70,12 @@ class ProjectMember < UserBasic
       end
     end
 
+    class ProjectMemberForAdmin < UserBasicForAdmin
+      expose :access_level do |user, options|
+        options[:project].project_members.find_by(user_id: user.id).access_level
+      end
+    end
```

Comment:
I rather inherited from `ProjectMember` and added email field. Otherwise its easy to make a mistake - add field to `ProjectMember` but not to `ProjectMemberForAdmin`